assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -66,7 +66,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This could be <code>### str.upper?</code>  Same to below. You can address this in a later PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/386/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>